### PR TITLE
Added Cc to header informations table and attachment names below the body text 

### PR DESCRIPTION
**Pull request by mattgolem**
Added Cc to header informations table.
Added attachment names below the body text (option "-aa", "--add-attachment-names").

### CODE REVIEW

**Comment by nickrussler** [comment]
Hi mattgolem, thanks for your contribution!

The code looks good to me. 

Please consider extending the unit tests, I would be grateful :)

**Comment by mattgolem** [comment]
unit tests added

**Comment by mattgolem** [comment]
Can you make new build?

File: src/main/java/mimeparser/MimeMessageConverter.java [hunk 1]
```diff
@@ -298,6 +297,18 @@ public String replace(Matcher m) throws Exception {
                     ADD_HEADER_IFRAME_JS_TAG_TEMPLATE, tmpHtmlHeader.toURI(), Resources.toString(contentScriptResource, StandardCharsets.UTF_8));
         }
 
+        // Append attachment filename list to body
+        if (addAttachmentNames) {
+            String attachmentsHtml = "";
+            List<AttachmentResource> attachments = EmailConverter.mimeMessageToEmail(message).getAttachments();
```
Comment (nickrussler) [inline src/main/java/mimeparser/MimeMessageConverter.java:303]: I wonder what the performance penalty for this is, but since it's behind a flag, it's ok.

**Comment by nickrussler** [comment]
Great stuff, thanks for your contribution.

**Comment by roboterwithsenf** [comment]
Thanks a lot!

Could you also update the setup.exe binary? As far as I see the feature is not yet implemented there.

**Comment by mattgolem** [comment]
The "innosetup.iss" script remains the same. What should I change?
Can make you the new binaries please?

**Pull request merged**